### ISSUE TITLE
[3.3.5a] Val'kyr Shadowguard are not spawning (25man)

### ISSUE THREAD

**Issue by selenir** [TrinityCore/TrinityCore#15048]
Not tested on 25 HC, only on NORM.

b574c99

They're just not spawning, no idea why.

**Comment by Keader** [comment]
25 hc (Lich King) not spawning too.

**Comment by Rushor** [comment]
you tested this with >5 people?

**Comment by Laintime** [comment]
Check carefully , everything works.
Not confirm.https://github.com/TrinityCore/TrinityCore/commit/a6f7bb029c8d29c19de8596d5d402222ca6c00a2

**Comment by Nyeriah** [comment]
They aren't spawning for me either.

**Comment by Keader** [comment]
@Rushor Until about four months ago they appeared even with one person in the raid.

**Comment by Pirricli** [comment]
confirm on b4a293fd346d622e25e30fca8fe5206be2da3745

**Comment by selenir** [comment]
Tested 3234 times with 25 players in a raid. They are not spawning. LK says: VALKYR BLAH BLAH and nothing happens.

**Comment by Frytiks** [comment]
I'm pretty sure it's cause so much latest DB updates. Somewhere is fuckup.

**Comment by Frytiks** [comment]
That bug is there more then 3 weeks

**Comment by selenir** [comment]
Also doing some testing, this is a hard one.

**Comment by Keader** [comment]
```diff

diff --git a/src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp b/src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp
index 755bbd7..c861450 100644
--- a/src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp
+++ b/src/server/scripts/Northrend/IcecrownCitadel/boss_the_lich_king.cpp
@@ -1005,7 +1005,7 @@ class boss_the_lich_king : public CreatureScript
                         case EVENT_SUMMON_VALKYR:
                             me->GetMap()->SetZoneMusic(AREA_THE_FROZEN_THRONE, MUSIC_SPECIAL);
                             Talk(SAY_LK_SUMMON_VALKYR);
-                            DoCastAOE(SUMMON_VALKYR);
+                            DoCastAOE(SUMMON_VALKYR,true);
                             events.ScheduleEvent(EVENT_SUMMON_VALKYR, urand(45000, 50000), 0, PHASE_TWO);
                             break;
                         case EVENT_START_ATTACK:

```

**Comment by Kittnz** [comment]
PR it please =)

**Comment by Keader** [comment]
@Kittnz I believe this will be fixed by #16322

**Comment by Demonid** [comment]
@Keader Already was using it, it works but it's a hackfix i guess.
